Pipfile

```diff
@@ -11,7 +11,6 @@ python_version = "3"
 constant-sorrow = ">=0.1.0a9"
 bytestring-splitter = ">=2.4.0"
 hendrix = ">=3.4"
-lmdb = "*"
```

Comment:
Should we relock dependencies? Or are we waiting until the code massacre is over 😄 , and then do a fresh relock?

Reply:
Yep - I'd like to perform a full relock in an isolated PR to avoid unrelated issues. 😅

Reply:
👌 Makes sense.